src/addrman.h

```diff
@@ -295,8 +310,17 @@ friend class CAddrManTest;
 
     /**
      * Serialized format.
-     * * version byte (@see `Format`)
-     * * 0x20 + nKey (serialized as if it were a vector, for backward compatibility)
+     * * format version byte (@see `Format`)
+     * * lowest compatible format version byte. This is used to help old software decide
+     *   whether to parse the file. For example:
+     *   * Bitcoin Core V knows how to parse up to format=3. If a new format=4 is introduced
```

Comment:
Bitcoin Core 5? These look like Roman numerals (anyone remember the Saturn V rocket?) 

Perhaps:
```diff
-     *   whether to parse the file. For example:
-     *   * Bitcoin Core V knows how to parse up to format=3. If a new format=4 is introduced
-     *     in Bitcoin Core V+1 which is compatible with format=3 and it is known that V will
-     *     be able to parse it, then V+1 will write (format=4, lowest_compatible=3) in the
-     *     first two bytes of the file and so V will still try to parse it.
-     *   * Bitcoin Core V+2 introduces a new incompatible format=5. It will write
+     *   whether to parse the file.
+     *   For example:
+     *   * Bitcoin Core version n knows how to parse up to format=3. If a new format=4 is introduced
+     *     in version n + 1 that is compatible with format=3 and it is known that version n will
+     *     be able to parse it, then version n + 1 will write (format=4, lowest_compatible=3) in the
+     *     first two bytes of the file, and so version n will still try to parse it.
+     *   * Bitcoin Core version n + 2 introduces a new incompatible format=5. It will write
```

Reply:
Will tweak if I retouch :)

Reply:
Reworded.

Reply:
I am somewhat partial to the idea of a roman numerals based version scheme :slightly_smiling_face: *upcoming release will be Bitcoin Core XXI* lol